solution/ui/regulations/eregs-vite/src/components/resources/ResourceExportBtn.vue

```diff
@@ -15,12 +24,17 @@
 <script>
 
 import exportFromJSON from 'export-from-json'
+import SimpleSpinner from "../../../../../regulations/js/src/components/SimpleSpinner.vue";
 import {
     getSupplementalContentV3,
 
 } from "../../utilities/api";
```

Comment:
Similarly you can import api as

`import { getSupplementalContentV3 } from "@/utilities/api";`